src/hooks/useModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactElement, useCallback, useContext, useEffect } from 'react';
+import { ModalContext } from '../context/ModalContext';
+import { IModalProps } from '../Alerts/atom/Modal';
+import { stringify } from 'querystring';
```

Comment:
🤦‍♀️ VS Code does it again. Sorry I'll be more careful.

Reply:
It's all good. It catches me out too from time to time. If it broke something it would be easier to spot but it always sneaks in when you're (often literally) not looking.